NewsApp/NewsApp/Model/Extension_String_+formattedDate.swift

```diff
@@ -0,0 +1,18 @@
+//
```

Comment:
La convention de nommage (de fichier) pour les extensions c'est String+FormattedDate.swift (par exemple si on fait une extension de String avec la fonctionnalité de formattage de date)

Reply:
fix in "renommage des extensions"